eventuate-tram-optimistic-locking/src/main/java/io/eventuate/tram/optimisticlocking/OptimisticLockingDecorator.java

```diff
@@ -0,0 +1,28 @@
+package io.eventuate.tram.optimisticlocking;
+
+import io.eventuate.tram.consumer.common.MessageHandlerDecorator;
+import io.eventuate.tram.consumer.common.MessageHandlerDecoratorChain;
+import io.eventuate.tram.consumer.common.SubscriberIdAndMessage;
+import org.springframework.core.Ordered;
+import org.springframework.orm.ObjectOptimisticLockingFailureException;
+import org.springframework.retry.annotation.Backoff;
+import org.springframework.retry.annotation.Retryable;
+
+import javax.transaction.Transactional;
```

Comment:
This should be Spring's `@Transactional`

Reply:
ok